src/relay/backend/compile_engine.cc

```diff
@@ -291,6 +308,12 @@ class CompileEngineImpl : public CompileEngineNode {
     auto spair = CreateSchedule(key->source_func, key->target);
     auto cache_node = make_node<CachedFuncNode>(
         *(spair.second.operator->()));
+    // Skip lowering for device copy node.
```

Comment:
I think we should make/have a set of "built-in"/"intrinsic" ops which we mark specially. For example I worry about name clash with user generated names, we should maybe implement a set of reserved names?

Reply:
@jroesch We can probably just have a map to contain the reserved builtin names and make them start with "__". But currently we only have `func_name` that starts with "fused" indicating the fused ops. I think user generated names should not affect here.

Reply:
This is a attribute for device_copy op. Let me see if we can just check that.